### DISCUSSION TITLE
LightBulb requires Windows Update

### DISCUSSION THREAD

**Discussion by Vanguger** [Tyrrrz/LightBulb#212] [General]
Why LightBulb Requires KB3063858 and KB3154518 Updates?
I have an installation error. After when I click Install, and after restarting Windows, they are still not installed. Everything was fine until today.

**Comment by Tyrrrz** [comment]
Same as https://github.com/Tyrrrz/LightBulb/issues/209?

**Reply by Vanguger** [reply]
Yes.

**Comment by Vanguger** [comment]
Button ignore, which constantly appears when loading, it is a crutch. I do not like this behavior of your application. I deleted your program. And you personally, Big Dislike for such an application behavior.
